staging/operator-registry/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/operator-framework/operator-registry
 
-go 1.15
+go 1.13
```

Comment:
What version of the registry are we grabbing here? I know master has been bumped to 1.15 and I removed the apiserver dependency in a recent PR.

Reply:
This is the latest tagged version as of now, v1.16.1. master tracking master could pull in unstable code to downstream master.

Reply:
I was under the impression we could simply pin against known release branches (e.g. release-4.8 in this case) instead of releases, but that method wouldn't scale to subsequent releases once the upstream/downstream split has happened and upstream has its own release cadence, so this method makes more sense to me.